packages/perseus-editor/src/widgets/__stories__/interactive-graph-editor.stories.tsx

```diff
@@ -66,151 +52,3 @@ export const Default: Story = {};
-                coord: [-1, -1],
-            },
-        ],
-    },
-    decorators: [StatefulDecorator],
-};
-
-/**
- * This InteractiveGraphEditor has a locked line segment, line, and ray.
- *
- * Locked figures are graph elements such as points, lines, line segements,
- * etc. that are locked in place and not interactive. They can be added
- * with the "Add element" dropdown at the bottom.
- */
-export const WithLockedLines: Story = {
-    args: {
-        // Use locked figures with mafs only.
-        ...mafsOptions,
-        lockedFigures: [
-            {
-                type: "line",
-                kind: "line",
-                points: [
-                    {...defaultPointProps, coord: [0, 2]},
-                    {...defaultPointProps, coord: [2, 3]},
-                ],
-                color: "green",
-                lineStyle: "solid",
-                showPoint1: false,
-                showPoint2: false,
-            },
-            {
-                type: "line",
-                kind: "ray",
-                points: [
-                    {...defaultPointProps, color: "pink", coord: [0, 0]},
-                    {...defaultPointProps, color: "pink", coord: [4, 2]},
-                ],
-                color: "pink",
-                lineStyle: "solid",
-                showPoint1: true,
-                showPoint2: false,
-            },
-            {
-                type: "line",
-                kind: "segment",
-                points: [
-                    {...defaultPointProps, color: "grayH", coord: [0, -2]},
-                    {...defaultPointProps, color: "grayH", coord: [4, 0]},
-                ],
-                color: "grayH",
-                lineStyle: "solid",
-                showPoint1: true,
-                showPoint2: true,
-            },
-        ],
-    },
-    decorators: [StatefulDecorator],
-};
-
-export const WithLockedEllipses: Story = {
-    args: {
-        // Use locked figures with mafs only.
-        ...mafsOptions,
-        lockedFigures: [
-            {
-                type: "ellipse",
-                center: [0, 0],
-                radius: [5, 2],
-                angle: 0,
-                color: "green",
-                fillStyle: "translucent",
-                strokeStyle: "solid",
-            },
-        ],
-    },
-    decorators: [StatefulDecorator],
-};
-
-export const WithLockedPolygons: Story = {
-    args: {
-        // Use locked figures with mafs only.
-        ...mafsOptions,
-        lockedFigures: [
-            {
-                type: "polygon",
-                points: [
-                    [-9, 4],
-                    [-6, 4],
-                    [-6, 1],
-                    [-9, 1],
-                ],
-                color: "green",
-                fillStyle: "translucent",
-                strokeStyle: "solid",
-                showVertices: true,
-            },
-        ],
-    },
-    decorators: [StatefulDecorator],
-};
```

Comment:
These locked figure-specific stories I added before felt out of place to me. They're already covered by different stories anyway, so I decided to remove them from this file.